### PR TITLE
removeTurret

### PR DESCRIPTION
**Pull request by Anuva12**

### CODE REVIEW

File: y2023/ChargedUp/src/main/java/com/gos/chargedup/RobotContainer.java [hunk 1]
```diff
@@ -236,22 +232,20 @@ private void configureBindings() {
...
+        //leftJoystickAsButtonLeft.whileTrue(m_turret.commandMoveTurretClockwise());
         leftJoystickAsButtonUp.whileTrue(m_armPivot.commandPivotArmUp());
         leftJoystickAsButtonDown.whileTrue(m_armPivot.commandPivotArmDown());
         m_operatorController.a().whileTrue(m_claw.createTeleopMoveClawIntakeInCommand(m_operatorController));
         m_operatorController.x().whileTrue(m_claw.createMoveClawIntakeOutCommand());
         m_operatorController.povUp().whileTrue(CombinedCommandsUtil.armToHpPickup(m_armPivot, m_armExtend));
-        m_operatorController.povDown().whileTrue(CombinedCommandsUtil.goToGroundPickup(m_armPivot, m_armExtend, m_turret));
-        m_operatorController.povLeft().whileTrue(CombinedCommandsUtil.goHome(m_armPivot, m_armExtend, m_turret));
-        //m_operatorController.povDown().whileTrue(CombinedCommandsUtil.armReset(m_armPivot, m_armExtend, m_turret));
+        //m_operatorController.povDown().whileTrue(CombinedCommandsUtil.goHome(m_armPivot, m_armExtend, m_turret));
```
Comment (pjreiniger) [inline y2023/ChargedUp/src/main/java/com/gos/chargedup/RobotContainer.java:246]: We still need `goHome`.

File: y2023/ChargedUp/src/main/java/com/gos/chargedup/autonomous/AutonomousFactory.java [hunk 2]
```diff
@@ -66,13 +65,13 @@ public AutonomousFactory(ChassisSubsystem chassis, TurretSubsystem turret, ArmPi
...
+
+            //m_autoOptions.get(height).put(AutonMode.ONE_NODE_AND_LEAVE_COMMUNITY_1, new OnePieceAndLeaveCommunityCommandGroup(chassis, armPivot, armExtension, claw, turret, "ONEPieceAndLeaveCommunity1", height, GamePieceType.CUBE));
+            //m_autoOptions.get(height).put(AutonMode.ONE_NODE_AND_LEAVE_COMMUNITY_7, new OnePieceAndLeaveCommunityCommandGroup(chassis, armPivot, armExtension, claw, turret, "ONEPieceAndLeaveCommunity7", height, GamePieceType.CUBE));
 
 
             // m_autoOptions.get(height).put(AutonMode.ONE_NODE_AND_ENGAGE_3, new OnePieceAndEngageCommandGroup(chassis, armPivot, armExtension, claw, "ONEPieceDockandEngage3", height, GamePieceType.CONE));
-            m_autoOptions.get(height).put(AutonMode.ONE_NODE_AND_ENGAGE_4, new OnePieceAndEngageCommandGroup(chassis, armPivot, armExtension, claw, turret, "ONEPieceDockandEngage4", height, GamePieceType.CUBE));
+            //m_autoOptions.get(height).put(AutonMode.ONE_NODE_AND_ENGAGE_4, new OnePieceAndEngageCommandGroup(chassis, armPivot, armExtension, claw, turret, "ONEPieceDockandEngage4", height, GamePieceType.CUBE));
```
Comment (pjreiniger) [inline y2023/ChargedUp/src/main/java/com/gos/chargedup/autonomous/AutonomousFactory.java:74]: Why were these removed?

File: y2023/ChargedUp/src/main/java/com/gos/chargedup/commands/CombinedCommandsUtil.java [hunk 3]
```diff
@@ -34,9 +34,9 @@ public static CommandBase armToHpPickup(ArmPivotSubsystem pivot, ArmExtensionSub
             .withName("HP Pickup");
     }
 
-    public static CommandBase moveToScore(double turretAngle, AutoPivotHeight height, GamePieceType gamePiece, TurretSubsystem turret, ArmPivotSubsystem armPivot) {
+    public static CommandBase moveToScore(double turretAngle, AutoPivotHeight height, GamePieceType gamePiece, ArmPivotSubsystem armPivot) {
```
Comment (pjreiniger) [inline y2023/ChargedUp/src/main/java/com/gos/chargedup/commands/CombinedCommandsUtil.java:37]: `turretAngle` is unnecessary

File: y2023/ChargedUp/src/main/java/com/gos/chargedup/autonomous/TwoPieceAndEngageCommandGroup.java [hunk 4]
```diff
@@ -33,4 +32,9 @@ public TwoPieceAndEngageCommandGroup(ChassisSubsystem chassis, TurretSubsystem t
         //drive
         addCommands(fullAuto);
     }
+
+    //initialie
+    //score piece
+    //chassis.rotate(90)
+    //runautopath
```
Comment (pjreiniger) [inline y2023/ChargedUp/src/main/java/com/gos/chargedup/autonomous/TwoPieceAndEngageCommandGroup.java:39]: Remove

**Review by pjreiniger — CHANGES_REQUESTED** [review]

**Review by ashley-wei — APPROVED** [review]

**Review by pjreiniger — APPROVED** [review]
